ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -0,0 +1,570 @@
+	'Power Infusion',
+);
+
+///////////////////////////////////////////////////////////////////////////
+//                                 WORLD BUFFS
+///////////////////////////////////////////////////////////////////////////
+
+export const RallyingCryOfTheDragonslayer = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(22888), fieldName: 'rallyingCryOfTheDragonslayer'}),
+	'Rallying Cry of the Dragonslayer',
+);
+
+export const SpiritOfZandalar = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(24425), fieldName: 'spiritOfZandalar'}),
+	'Spirit of Zandalar',
+);
+export const SongflowerSerenade = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(15366), fieldName: 'songflowerSerenade'}),
+	'Songflower Serenade',
+);
+export const WarchiefsBlessing = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(16609), fieldName: 'warchiefsBlessing'}),
+	`Warchief's Blessing`,
+);
+
+export const SaygesDarkFortune = makeEnumIndividualBuffInput({
+	numColumns: 6,
+	direction: IconEnumPickerDirection.Horizontal,
+	values: [
+		{ iconUrl: 'https://wow.zamimg.com/images/wow/icons/large/inv_misc_orb_02.jpg', value: SaygesFortune.SaygesUnknown, text: `Sayge's Dark Fortune` },
+		{ id: ActionId.fromSpellId(23768), value: SaygesFortune.SaygesDamage, text: `Sayge's Damage` },
+		{ id: ActionId.fromSpellId(23736), value: SaygesFortune.SaygesAgility, text: `Sayge's Agility` },
+		{ id: ActionId.fromSpellId(23766), value: SaygesFortune.SaygesIntellect, text: `Sayge's Intellect` },
+		{ id: ActionId.fromSpellId(23738), value: SaygesFortune.SaygesSpirit, text: `Sayge's Spirit` },
+		{ id: ActionId.fromSpellId(23737), value: SaygesFortune.SaygesStamina, text: `Sayge's Stamina` },
+	],
+	fieldName: 'saygesFortune'
+})
+
+// Dire Maul Buffs
+export const FengusFerocity = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(22817), fieldName: 'fengusFerocity'}),
+	`Fengus' Ferocity`,
+);
+export const MoldarsMoxie = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(22818), fieldName: 'moldarsMoxie'}),
+	`Moldar's Moxie`,
+);
+export const SlipKiksSavvy = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(22820), fieldName: 'slipkiksSavvy'}),
+	`Slip'kik's Savvy`,
+);
+
+// SoD World Buffs
+export const BoonOfBlackfathom = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(430947), fieldName: 'boonOfBlackfathom'}),
+	'Boon of Blackfathom',
+);
+
+///////////////////////////////////////////////////////////////////////////
+//                                 DEBUFFS
+///////////////////////////////////////////////////////////////////////////
+
+export const MajorArmorDebuff = InputHelpers.makeMultiIconInput([
+	makeBooleanDebuffInput({id: ActionId.fromSpellId(11597), fieldName: 'sunderArmor'}),
+	makeTristateDebuffInput(ActionId.fromSpellId(11198), ActionId.fromSpellId(14169), 'exposeArmor'),
+], 'Armor Penetration');
```

Comment:
Would probably keep the old name of "Major ArP" or equivalent here, since CoR and FF below are also "Armor Penetration" debuffs, just minor ones that can be stacked unlike the ones in this category.